src/utils/getUniqueIdGenerator.spec.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env mocha */
+import {assert} from 'chai';
+import getUniqueIdGenerator from './getUniqueIdGenerator';
+
+describe('utils/getUniqueIdGenerator', () => {
+  let gen;
+  let anotherGen;
+
+  beforeEach(() => {
+    gen = getUniqueIdGenerator();
+    anotherGen = getUniqueIdGenerator();
+  });
+
+  describe('one generator', () => {
+    it('should generate different uniqueIds each time it is invoked', () => {
+      assert.notEqual(gen(), gen());
```

Comment:
@oliviertassinari The generated ids are used as html tag attributes in most cases, which may contain an implicit type conversion.

```javascript
const id1 = 1;
const id2 = '1';
document.head.id = id1;
document.body.id = id2;
console.log(id1 === id2); // false
console.log(document.head.id === document.body.id); // true
```

`id1` and `id2` will pass `notStrictEqual`. But `document.head` and `document.body` will have conflicts using same id.